packages/common/src/types/proof.ts

```diff
@@ -46,7 +46,9 @@ export type TMinaSignature = {
   data: string;
 };
 
-export type TZkDatabaseProof = JsonProof;
+export type TZkDatabaseProof = JsonProof & {
```

Comment:
This type should be moved to `application` and impot `TRollupSerializedProof` from `@zkdb/smart-contract`